lib/shared-method.js

```diff
@@ -274,7 +272,7 @@ SharedMethod.prototype.invoke = function(scope, args, remotingOptions, ctx, cb)
     }
     return retval;
   } catch (err) {
-    debug('error caught during the invocation of %s', this.name);
+    debug('error caught during the invocation of %s', this.name, err);
```

Comment:
Good catch. `debug` prints any extra arguments the same way as `console.log`, but then the timing info is inserted int the middle of the message. I'll fix this.